jetty-server/src/main/java/org/eclipse/jetty/server/Request.java

```diff
@@ -1322,20 +1321,20 @@ public String getScheme()
     @Override
     public String getServerName()
     {
-        return _uri == null ? findServerName() : _uri.getHost();
+        return _uri == null ? findServerName() : formatAddrOrHost(_uri.getHost());
```

Comment:
We were not normalising before here and now we are. So not sure if this will impact anything.

Reply:
The previous impl was inconsistent as we normalise in `findServerName()` but not in `_uri.getHost()`, although the later is probably already normalised.  I think this is a low risk change for the better.